Services/JobService.cs

```diff
@@ -70,22 +80,42 @@ public async Task GenerateAndSendRecipes(string familyId)
     {
         // grab family, get gordon's prompt, create the email, then send it
         var family = _familyService.GetById(familyId).Data;
-        var gordonPrompt = BuildGordonPrompt(family!)!;
-        var gordonResponse = await _gordonService.GetMessageResponse(gordonPrompt);
-        var body = await _viewToStringService.ViewToStringAsync(
-            "EmailTemplate",
-            gordonResponse.Data!
-        );
-
-        if (family != null && body != null)
+        if (family != null)
```

Comment:
should we change this to a check if family is null and if so throw an error? otherwise just proceed with the rest of the code? currently, we aren't handling the case were family or body (below) is null

Reply:
Thats a good catch, I will update those two paths to throw errors